MAUI/ListView/working-with-sflistview.md

```diff
@@ -0,0 +1,769 @@
+ * Avoid loading complex layout in template of listview. For example, loading large-size images or nested containers degrades the scrolling performance. This practice commonly degrades performance in all platforms, and particularly more in Android version API level 19. So, use fewer elements and images with small size and resolution to achieve the maximum performance.
+ * Avoid placing the `SfListView` inside `ScrollView` for the following reasons:
+    * The `SfListView` implement its own scrolling.
+    * The `SfListView` will not receive any gestures as it will be handled by the parent `ScrollView`.
+    * Should define size to the `SfListView` if it loads inside `ScrollView`.
+ * Avoid changing the cell layout based on the `BindingContext`. This incurs large layout and initialization costs.
+ * Implement a model class inherited with `INotifyPropertyChanged` interface to notify the property changes at runtime.
+
+ ## Scrolling ListView without virtualization
+
+`SfListView` allows you to scroll by loading the entire collection of items inside the `ScrollView` and defining the total extend of its container to `HeightRequest` in the [Loaded](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_Loaded) event.
+
+{% tabs %}
+{% highlight xaml %}
+<ScrollView>
+   <sync:SfListView x:Name="listView" ItemsSource="{Binding BookInfo}" Loaded="listView_Loaded"/>
+</ScrollView>
+{% endhighlight %}
+{% highlight C# %}
+using Syncfusion.ListView.Control.Helpers;
+public partial class MainPage : ContentPage
+{ 
+	public MainPage()
+	{
+		InitializeComponent();
+	}
+ 
+	private void listView_Loaded(object sender,  e)
+	{
+		var container = listView.GetVisualContainer();
+		var extent = (double)container.GetType().GetRuntimeProperties().FirstOrDefault(x => x.Name == "TotalExtent").GetValue(container);
+		listView.HeightRequest = extent;
+	}
+}
+{% endhighlight %}
+{% endtabs %}
+
+
+When `SfListView` is in [AutoFitMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_AutoFitMode) as `Height`, the extend of the ListView will be updated only while scrolling. So you can resize the ListView in [VisualContainer]((https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.VisualContainer.html)) `PropertyChanged` method as like below.

```

Comment:
So you can resize the ListView in [VisualContainer]((https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.VisualContainer.html)) `PropertyChanged` method, as shown in the following code sample.

Reply:
I have modified the line as per the concerns provided